app/controller_services/shopping_lists_controller/destroy_service.rb

```diff
@@ -35,6 +39,14 @@ def shopping_list
       @shopping_list ||= user.shopping_lists.find(list_id)
     end
 
+    def aggregate_list
+      game.aggregate_shopping_list
```

Comment:
It's not an accident that I didn't include `@aggregate_list ||=` here - memoising the value actually would precent ActiveRecord from detecting that the list wasn't persisted on line 24.